### PR TITLE
add icml talk info

### PR DESCRIPTION
**Pull request by suzhoum**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AWESOME.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ To get started, we recommend watching [AutoGluon 1.0: Shattering the AutoML Ceil
 
 | Title                                                                                                                                                              | Format   | Location                                                                                                                           | Date       |
 |--------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------|------------|
+| [AutoGluon: AutoML at Your Fingertips](https://suzhoum.github.io/icml-24-autogluon-talk/)                                                                | Expo Talk | [ICML 2024](https://icml.cc/Expo/Conferences/2024/talk%20panel/36234)                                                            | 2024/07/21 |
```
Comment (Innixma) [inline AWESOME.md:18]: The slides should be made public, currently I can't access them.

Fingertip -> Fingertips

**Review by Innixma — COMMENTED** [review]
Thanks for giving the talk and sharing the link! I added a comment

File: AWESOME.md (hunk 1 above)
Comment (tonyhoo) [reply AWESOME.md:18]: Same here, I think the link is only accessible to people registered. Would be good to upload it to our team public S3 bucket

Comment (suzhoum) [reply AWESOME.md:18]: I updated the permission. Could you please check again?

Comment (Innixma) [reply AWESOME.md:18]: I can see them now, thanks @suzhoum !

**Review by Innixma — APPROVED** [review]

**Pull request merged**